project-generated-code-backend/Backend/Model/Accounts/Patient.cs

```diff
@@ -27,6 +27,11 @@ public Patient(string name, string surname, string id, DateTime dateOfBirth, str
             this.password = password;
         }
 
+        public Patient(string name, string surname, string id) : base(Guid.NewGuid().ToString(), name, surname, id)
+        {
+
+        }
```

Comment:
Instead of generating new guid here, you should add a constructor in Account class that accepts name, surname and id, but calls empty base constructor so that it generates guid there (in the entity class).